docs/UserGuide.md

```diff
@@ -16,17 +16,17 @@ management applications.
    2. [Listing all persons](#listing-all-persons--listp)
    3. [Listing all concerts](#listing-all-concerts--listc)
    4. [Listing all persons and concerts](#listing-all-persons-and-concerts--list)
-   5. [Listing all concertContacts](#listing-all-concertcontacts--listcc)
+   5. [Listing all concert contacts](#listing-all-concertcontacts--listcc)
    6. [Adding a Person](#adding-a-person-addp)
    7. [Adding a Concert](#adding-a-concert-addc)
-   8. [Adding a ConcertContact](#adding-a-concertcontact-addcc)
+   8. [Adding a Concert Contact](#adding-a-concertcontact-addcc)
```

Comment:
Need to be consistent the above concert contact not caps the ones below not caps too